src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1165,6 +1165,8 @@ public override void Execute(ICpu cpu)
             if (Direct)
             {
                 functionPointer = cpu.GetValue(Destination);
+                if (functionPointer == null)
+                    throw new KOSException("Attempt to call function failed - Value of function pointer for " + Destination + " is null.");
```

Comment:
This is one of those "the user should never see this message" types of errors.  I added it to help me force a crash right at the spot where the problem first occurred, so I could get the dump of what the stack was like at that point.  But I decided to leave it in, because if this ever DOES happen, well, its really not possible without some kind of kOS bug and I'd like to know about it.